### PR TITLE
Discovered that we had too much randomness for our concrete users.

### PR DESCRIPTION
**Pull request by charlesstrange2**
Reduced that by fixing some centres into the right sponsors.

### CODE REVIEW

File: database/seeds/CentreUsersSeeder.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ public function run()
         ]);
 
         // Get the first centre's sponsor and make two more centres with the same sponsor
-        $sponsor_id = $user1->centres()->first()->sponsor()->first()->id;
+        $sponsor_id = $user1->centres()->first()->sponsor->id;
```
Comment (katjam) [inline database/seeds/CentreUsersSeeder.php]: Makes more sense

File: database/seeds/CentreUsersSeeder.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ public function run()
             "password" => bcrypt('store_pass'),
             "role" => "foodmatters_user",
         ]);
-        $user3->centres()->attach(1, ['homeCentre' => true]);
+        $user3->centres()->attach(2, ['homeCentre' => true]);
```
Comment (katjam) [inline database/seeds/CentreUsersSeeder.php]: I'm guessing there is a good reason to go for second centre.

File: database/seeds/CentresSeeder.php [hunk 3]
```diff
@@ -11,15 +11,21 @@ class CentresSeeder extends Seeder
...
-        $centres = factory(App\Centre::class, 4)->create();
+        $centres = factory(App\Centre::class, 3)->create();
 
         // Grab one and change print pref to individual and another to collection.
         $centres[0]->print_pref = 'collection';
         $centres[0]->save();
         $centres[1]->print_pref = 'individual';
         $centres[1]->save();
 
+
```
Comment (katjam) [inline database/seeds/CentresSeeder.php]: lots of extra whiteness

File: database/seeds/CentreUsersSeeder.php (hunk 2 above)
Comment (charlesstrange2) [reply database/seeds/CentreUsersSeeder.php]: Yes, the second centre gets a different print_pref, and at least one user ought to have it so karl cna test.

**Pull request merged**